draft-ietf-oauth-transaction-tokens.md

```diff
@@ -591,6 +610,9 @@ The `expires_in` and `scope` fields of the OAuth 2.0 Token Exchange specificatio
 ## Access Tokens
 When creating Txn-Tokens, the Txn-Token MUST NOT contain the Access Token presented to the external endpoint. If an Access Token is included in a Txn-Token, an attacker may extract the Access Token from the Txn-Token, and replay it to any Resource Server that can accept that Access Token. Txn-Token expiry does not protect against this attack since the Access Token may remain valid even after the Txn-Token has expired.
 
+## Subject Token Types {#sec-sub-token-types}
+A service requesting a Txn-Token SHOULD provide an incoming token if it has one that it used itself to authorize a caller, and if it directly correlates with the downstream call chain it needs the Txn-Token for. In the absence of an appropriate incoming token, the requesting service MAY use a self-signed JWT, an unsigned JSON object or any other format to represent the requester to the Txn-Token service.
```

Comment:
To represent the requester? Maybe "to express the properties it desires from the generated Txn-Token".